plugins/GeoIp2/lang/en.json

```diff
@@ -1,6 +1,8 @@
 {
   "GeoIp2": {
     "AssumingNonApache": "Cannot find apache_get_modules function, assuming non-Apache webserver.",
+    "AutomaticSetup": "Automatically configure geolocation using a dbip database",
+    "AutomaticSetupDescription": "For a proper geolocation Matomo requires an external database. Using this option, Matomo will automatically be configured to download an use the latest %1$sdbip%2$s city level database.",
```

Comment:
could maybe say `... city level database which is licensed under the [Creative Commons Attribution 4.0 International License](https://db-ip.com/db/lite.php?refid=mtm)".` or just say `... city level database. [View licensing terms](https://db-ip.com/db/lite.php?refid=mtm)`. Not sure we do this in the GeoIP downloader already. If not, then it might not be too important. Generally be good though if we do it in the admin as well. fyi @Findus23

Reply:
![image](https://user-images.githubusercontent.com/273120/81747281-ef41f200-94fb-11ea-9b8a-656f3f669aca.png)
 sorry just seeing this is actually already done just not as clear.